assets/olive-optimizer/general/environments/gpu/context/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.6-cudnn8-ubuntu20.04:{{latest-image-tag}}

```

Comment:
=====================================================================================
  Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)
  
  ┌────────────────┬────────────────┬──────────┬───────────────────┬───────────────┬────────────────────────────────────────────┐
  │    Library     │ Vulnerability  │ Severity │ Installed Version │ Fixed Version │                   Title                    │
  ├────────────────┼────────────────┼──────────┼───────────────────┼───────────────┼────────────────────────────────────────────┤
  │ linux-libc-dev │ CVE-2023-35788 │ HIGH     │ 5.4.0-152.169     │ 5.4.0-153.170 │ out-of-bounds write in fl_set_geneve_opt() │
  │                │                │          │                   │               │ https://avd.aquasec.com/nvd/cve-2023-35788 │
  └────────────────┴────────────────┴──────────┴───────────────────┴───────────────┴────────────────────────────────────────────┘
  
  Python (python-pkg)
  ===================
  Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 0)
  
  ┌─────────────────────────┬────────────────┬──────────┬───────────────────┬───────────────┬────────────────────────────────────────────┐
  │         Library         │ Vulnerability  │ Severity │ Installed Version │ Fixed Version │                   Title                    │
  ├─────────────────────────┼────────────────┼──────────┼───────────────────┼───────────────┼────────────────────────────────────────────┤
  │ cryptography (METADATA) │ CVE-2023-38325 │ MEDIUM   │ 41.0.0            │ 41.0.2        │ cryptography mishandles SSH certificates   │
  │                         │                │          │                   │               │ https://avd.aquasec.com/nvd/cve-2023-38325 │
  └─────────────────────────┴────────────────┴──────────┴───────────────────┴───────────────┴────────────────────────────────────────────┘